aws-android-sdk-cognitoidentityprovider/src/main/java/com/amazonaws/mobileconnectors/cognitoidentityprovider/continuations/MultiFactorAuthenticationContinuation.java

```diff
@@ -67,6 +71,27 @@ public MultiFactorAuthenticationContinuation(CognitoUser user,
         this.callback = callback;
         this.runInBackground = runInBackground;
         this.challenge = challenge;
+        this.clientMetadata = Collections.emptyMap();
+    }
+
+    /**
+     * <p>
+     * <code>clientMetadata</code> is a map of custom key-value pairs that you can provide as input for any
+     * custom work flows. Accessor method for <code>clientMetadata</code>.
+     * </p>
+     * @return ClientMetadata
+     */
+    public Map<String, String> getClientMetaData() {
```

Comment:
Same as above two comments - it should be Metadata like you said but decided to leave this inconsistency since it was introduced in the already released version and I wanted the methods on these other Continuation classes to be consistent with that.